### DISCUSSION TITLE
Migration of Flow with node.js

### DISCUSSION THREAD

**Discussion by bjochelle** [directus/directus#21273] [Feature Requests]
### Summary

Just like the syncing with schema migrations using endpoints. Possible a replication of flow from one endpoint to another.

### Motivation

Easier Maintenance which can make it easier to find solutions to issues and keep the project up-to-date from one endpoint to another. Possible expected outcome is like the replication of schema

### Detailed Design

GET /flow/snapshot
GET /flow/diff
POST /flow/apply

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

*

### Alternatives

A long method to check which flow is updated, newly created or deleted

### Adoption Strategy

*

**Comment by rijkvanzanten** [comment]
Lets keep the discussion on duplicating / migration flows over in the existing discussion on the topic: https://github.com/directus/directus/discussions/14405

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
